backend/app/api/routes/model_management.py

```
@@ -0,0 +1,263 @@
+"""
+Model Management API routes for LiteLLM hot reload functionality.
+支持动态添加、删除、更新模型配置而无需重启服务。
+"""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)